### ISSUE TITLE
crash on kick seems to not even work, 

### ISSUE THREAD

**Issue by spiffyhax** [nullworks/cathook#1606]
It has rarely done it, and im not sure why

**Comment by Imortal-Wiso** [comment]
game probably had an update that messed with how voting works, it's that or the code for it is not up to date.

**Comment by STG7754** [comment]
Crash on kick cant enable on TEXTMODE

**Comment by b9017955** [comment]
That's good 😌

**Comment by BenCat07** [comment]
> Crash on kick cant enable on TEXTMODE

that is false, it just seems to have broken at some point. We check the disconnect message, and it seems to have changed already quite a while ago

**Comment by StuckInThePast** [comment]
@Taureon Y ou should change your name from Taureon to 'Tard.
